src/cordova/apple/xcode/macos/Outline.xcodeproj/project.pbxproj

```diff
@@ -633,6 +633,7 @@
 				COPY_PHASE_STRIP = NO;
 				ENABLE_STRICT_OBJC_MSGSEND = YES;
 				ENABLE_TESTABILITY = YES;
+				EXCLUDED_ARCHS = arm64;
```

Comment:
Please revert the other files. I believe you opened the project file instead of the workspace file.

Reply:
The workspace refers to the project. The workspace is needed when using Swift packages or cocoapods. The project can't refer to them. Or at least couldn't, now it seems they add an embedded workspace to the project, but Cordova doesn't support that.

As a rule of thumb, never open the project, always open the workspace.